src/main/java/de/adorsys/sdjwt/vp/SdJwtVP.java

```diff
@@ -84,7 +84,11 @@ public static SdJwtVP of(String sdJwtString) {
         int disclosureEnd = sdJwtString.lastIndexOf(SdJwt.DELIMITER);
 
         String issuerSignedJWTString = sdJwtString.substring(0, disclosureStart);
-        String disclosuresString = sdJwtString.substring(disclosureStart + 1, disclosureEnd);
+        String disclosuresString = "";
```

Comment:
You'll have a similar problem here if the search on L83 is unsuccessful, returning -1.

Reply:
Okay, Let me got through it.